src/v/kafka/server/connection_context.h

```diff
@@ -399,6 +399,13 @@ class connection_context final
         ss::lowres_clock::time_point _last_request_timestamp;
     };
 
+    struct throttling_state {
```

Comment:
Does it make sense for `_snc_quota_context` to be in here, too?

Reply:
follow up: are there plans to expand the contents of this struct?

Reply:
> follow up: are there plans to expand the contents of this struct?

No plans to expand the struct so I've just moved the timestamp to be a top-level field to keep things simple.